src/main/java/com/mixpanel/android/viewcrawler/ViewVisitor.java

```diff
@@ -204,73 +204,42 @@ public LayoutUpdateVisitor(List<Pathfinder.PathElement> path, LayoutRule args,
 
         @Override
         public void cleanup() {
-            for (Map.Entry<View, LayoutRule> original:mOriginalValues.entrySet()) {
+            // TODO find a way to optimize this.. remove this visitor and trigger a re-layout??
```

Comment:
You can probably remove this TODO - It looks like this cleanup is ok just as it is :)